catalog/plugin/catalog-plugin-query-monitor-commands/src/main/java/ddf.catalog.querymonitorcommands/QueryMonitorCommandRemoveFromTable.java

```diff
@@ -0,0 +1,96 @@
+
+public class QueryMonitorCommandRemoveFromTable implements Action {
+    @Argument(name = "trueOrFalse", description = "The boolean value indicating whether to remove activeSearches from the search table after the search has completed.", index = 0, multiValued = false, required = true)
+    private String trueOrFalse = null;
+
+    private static final Logger LOGGER =
+            LoggerFactory.getLogger(QueryMonitorCommandRemoveFromTable.class);
+
+    QueryMonitorPlugin queryMonitor1 = null;
+
+    public void setQueryMonitor1(QueryMonitorPlugin qmpi) {
+        queryMonitor1 = qmpi;
+    }
+
+    public QueryMonitorPlugin getQueryMonitor1() {
+        return queryMonitor1;
+    }
+
+    public QueryMonitorCommandRemoveFromTable(QueryMonitorPlugin q) {
+        queryMonitor1 = q;
+    }
+
+    public QueryMonitorCommandRemoveFromTable() {
+
+    }
+
+    /**
+     * Method to set the boolean in a {@link QueryMonitorPlugin} which determines whether {@link ddf.catalog.querymonitor.ActiveSearch}'s should be removed from the
+     * ActiveSearchTable when the search completes. A value of true has the effect that searches will be removed
+     * while a value of false has the effect that {@link ddf.catalog.querymonitor.ActiveSearch}'s will be left inside of the table after query completion.
+     *
+     * @param b {@link Boolean} a value of true has the effect that searches will be removed
+     *          while a value of false has the effect that {@link ddf.catalog.querymonitor.ActiveSearch}'s will be left inside of the ActiveSearchTable after query completion.
+     * @return {@link Boolean} representing whether or not setting the variable was successful
+     */
+    public boolean setRemoveFromTable(boolean b) {
+        if (queryMonitor1 == null) {
+            LOGGER.warn("QueryMonitor not yet instantiated! Cannot setRemoveFromTable(" + b + ")!");
+            return false;
+        }
+        queryMonitor1.setRemoveFromTable(b);
+        return true;
+    }
+
+    /**
+     * Method to implement {@link Action}. Called when the catalog:setremovefromtable command is called by the shell.
+     * Sets the value of the boolean which determines whether or not {@link ddf.catalog.querymonitor.ActiveSearch}'s are removed from the
+     * ActiveSearchTable after search completion.
+     *
+     * @return always returns {@code null}
+     */
+    @Override
+    public Object execute() throws Exception {
+        if (queryMonitor1 == null) {
+            LOGGER.debug("QueryMonitor not yet instantiated!");
+            return null;
+        }
+        if (trueOrFalse != null) {
+            if (trueOrFalse.toLowerCase()
+                    .equals(("true"))) {
+                queryMonitor1.setRemoveFromTable(true);
+            } else if (trueOrFalse.toLowerCase()
+                    .equals("false")) {
+                queryMonitor1.setRemoveFromTable(false);
+            } else {
+                LOGGER.warn(
```

Comment:
Might be more helpful to print something to the console so the user can immediately see what they did wrong.